internal/commands/user/disable.go

```diff
@@ -0,0 +1,121 @@
+package user
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/spf13/pflag"
+)
+
+// CommandDisable is the `user disable` command
+type CommandDisable struct {
+	inputs      disableInputs
+	outputs     []userOutput
+	realmClient realm.Client
+}
+
+type disableInputs struct {
+	app.ProjectInputs
+	Users []string
+}
+
+func (i *disableInputs) Resolve(profile *cli.Profile, ui terminal.UI) error {
+	if err := i.ProjectInputs.Resolve(ui, profile.WorkingDirectory); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (i *disableInputs) resolveUsers(ui terminal.UI, client realm.Client, app realm.App) ([]realm.User, error) {
+	filter := realm.UserFilter{
+		IDs: i.Users,
+	}
+	foundUsers, usersErr := client.FindUsers(app.GroupID, app.ID, filter)
+	if usersErr != nil {
+		return nil, usersErr
+	}
+	return foundUsers, nil
+}
+
+// Flags is the command flags
+func (cmd *CommandDisable) Flags(fs *pflag.FlagSet) {
+	cmd.inputs.Flags(fs)
+	fs.StringSliceVarP(&cmd.inputs.Users, flagUser, flagUserShort, []string{}, flagUserDisableUsage)
+}
+
+// Inputs is the command inputs
+func (cmd *CommandDisable) Inputs() cli.InputResolver {
+	return &cmd.inputs
+}
+
+// Setup is the command setup
+func (cmd *CommandDisable) Setup(profile *cli.Profile, ui terminal.UI) error {
+	cmd.realmClient = profile.RealmAuthClient()
+	return nil
+}
+
+// Handler is the command handler
+func (cmd *CommandDisable) Handler(profile *cli.Profile, ui terminal.UI) error {
+	app, err := app.Resolve(ui, cmd.realmClient, cmd.inputs.Filter())
+	if err != nil {
+		return err
+	}
+
+	users, usersErr := cmd.inputs.resolveUsers(ui, cmd.realmClient, app)
```

Comment:
I don't think we need the extra function here, this could just be:

```go
users, usersErr := client.FindUsers(app.GroupID, app.ID, realm.UserFilter{IDs: cmd.inputs.Users})
```